README.md

```diff
@@ -278,6 +278,30 @@ Basically, it will look like this:
   You can specify a class for the separator (which is wrapped by a `<span>` tag),
   if you want to use special design or interaction on it.
 
+## Cache
+
+If you want to cache your cms results, just activate it via the config:
+
+```yml
+    cache:
+        enabled: true
+        ttl: 300
+```
+
+It uses Doctrine Result Cache so you need to activate it to:
+
+```yml
+    doctrine:    
+        orm:
+            result_cache_driver: apc            
+```
+
+You can read more about DoctrineCache <a href="http://symfony.com/doc/current/reference/configuration/doctrine.html#caching-drivers">here</a>.
+
+
+
+
+
```

Comment:
Too many spaces here, could you reduce to one single space? 
There's a title the line below so it's already separated ;)